package.json

```diff
@@ -52,6 +52,8 @@
     "check-lazy-loading": "ts-node scripts/checkLazyLoading.ts",
     "lint-watch": "npx eslint-watch --watch --changed",
     "shellcheck": "./scripts/shellCheck.sh",
+    "spell": "cspell --no-progress --color **/*",
+    "spell-changed": "cspell --no-progress --color --no-must-find-files",
```

Comment:
NAB I kind of like progress enabled by default on both of these. Pretty nice to be able to get some visual feedback that's somethings happening, and disable it only if you want to.